experimental/class-features.md

```diff
@@ -0,0 +1,67 @@
+
+# Class Features
+
+This experimental extension covers three class features proposals: 
+[Class Fields], [Static Class Features] and [Private Methods].
+
+## ClassBody
+
+```js
+extend interface ClassBody {
+    body: [ MethodDefinition | PropertyDefinition ];
+}
+```
+
+## PropertyDefinition
+
+```js
+interface PropertyDefinition <: Node {
+    type: "PropertyDefinition";
+    key: Expression | PrivateName;
+    value: Expression | null;
+    computed: boolean;
+    static: boolean;
+    private: boolean;
+}
+```
+
+- When `private` is `true`, `computed` must be `false` and `key` must be a `PrivateName`.
+- When `private` is `false`, `key` must be an `Expression` and can not be a `PrivateName`.
+
+## MethodDefinition
+
+```js
+extend interface MethodDefinition {
+    key: Expression | PrivateName;
+    private: boolean;
+}
+```
+
+- When `private` is `true`, `computed` must be `false`, `key` must be a `PrivateName`, `kind` can not be `"constructor"`.
+- When `private` is `false`, `key` must be an `Expression` and can not be a `PrivateName`.
+
+### PrivateName
+
+```js
+interface PrivateName <: Node {
```

Comment:
Maybe we should use a different name because Babel's AST has a node which is the same name but a different shape. 
 How about [PrivateIdentifier](https://tc39.es/proposal-class-fields/#prod-PrivateIdentifier) as same as the language spec's token name?

Reply:
`PrivateIdentifier` sounds good to me.